pkg/k8s/event_handlers.go

```diff
@@ -29,11 +36,19 @@ func GetKubernetesEventHandlers(informerName, providerName string, shouldObserve
 
 	return cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
+			log := log.With().Str("event", eventTypes.Add.String()).Logger()
+			o, err := meta.Accessor(obj)
+			if err != nil {
+				log.Error().Err(err).Msg(msgErrParsingObject)
+				return
+			}
+			log = logWithResourceName(log, o)
```

Comment:
> I don't think we should log resources that are not being observed, it will create a lot of noise in larger environments with a lot of resources.

Done

> Can we move this snippet that logs the resource name observed and the event type and merge it with the `msgObservedEvent` log?

This is doing what I think you want it to. `log := log.With().Str("event", eventTypes.Add.String()).Logger()` isn't writing any logs itself, but effectively saying "create a new logger that includes an `event` field with this value" so then future `log.Msg()` calls all have that field.The new logger is local to the function so it's not modifying the package-scoped one. The resource observed message will include the event kind and name of the resource in the same line.